src/server/identity/server/api_server.go

```diff
@@ -53,8 +57,13 @@ func (a *apiServer) EnvBootstrap(ctx context.Context) error {
 	}
 	if a.env.Config.IdentityConnectors != "" {
 		a.env.Logger.Info("Started to configure identity connectors via environment")
-		var connectors []identity.IDPConnector
-		if err := yaml.Unmarshal([]byte(a.env.Config.IdentityConnectors), &connectors); err != nil {
+		var connectors identityutil.IDPConnectors
+		jsonBytes, err := yaml.YAMLToJSON([]byte(a.env.Config.IdentityConnectors))
```

Comment:
this still works if the src is JSON?

Reply:
Yup, its a no-op if the source is JSON. https://github.com/ghodss/yaml/blob/d8423dcdf3440d0a5baffc6f90a11e4128545620/yaml.go#L105